pkg/sdk/logging/api/v1beta1/fluentd_types.go

```diff
@@ -148,8 +148,10 @@ type FluentdTLS struct {
 type FluentdDrainConfig struct {
 	// Should buffers on persistent volumes left after scaling down the statefulset be drained
 	Enabled bool `json:"enabled,omitempty"`
-	// Container image to use for the drain watch sidecar
+	// Annotations to use for the drain watch sidecar
 	Annotations map[string]string `json:"annotations,omitempty"`
+	// Labels to use for the drain watch sidecar
```

Comment:
```suggestion
	// Labels to use for the drain watch sidecar on top of labels added by the operator by default
```

Reply:
Currently, users can also override the default labels. If that is the right behaviour, we should also add that to the comment, otherwise i should change the logic.

Reply:
yes, my intention was that, feel free to change the comment or ignore, it's just a nit